### ISSUE TITLE
Make new member welcome email more generic

### ISSUE THREAD

**Issue by Sandplum** [TampereHacklab/mulysa#505]
New members receive an email when their applications are accepted.  The email includes an invitation to attend meetings:

> Welcome aboard {{first_name}}!
> 
> Great to have you onboard! Your membership application has been
> approved and now is a great time to to join our community by
> coming to the next community meeting on the first Tuesday of the
> month or poping by weekly open house every Tuesday. It is also
> highly recommendable to join our communication channels (Matrix,
> Telegram or IRC and email list). Be happy to ask if you like to
> know more!

<br>
The message could be modified to be more generic, and not limited to organizations that have meetings on Tuesdays.
<br>
<br>

> Your membership application has been approved and now is a great time to to join our community by
> coming to the next community meeting or popping by during an open house event.

<br>

> It is also highly recommended to join our communication channels (Matrix and email list). Feel free to ask if you would like to
> know more!

The note regarding the bank sync could be updated to reflect actual sync schedule.

> Important note about door access! The payment data from bank is synced only once a day and this might
> mean that your payment is not seen by the system until the next sync happens. You can check when the last sync happened in {{siteurl}}

**Comment by olmari** [comment]
Related Issue (kinda prerequisite, not really but practically): https://github.com/TampereHacklab/mulysa/issues/172

**Comment by braaar** [comment]
I like the idea of making the email much more generic to begin with.

That will force those who want more specificity to do something like #172

**Comment by Sandplum** [comment]
Agreed.  As this issue is currently having a negative effect on one particular organization, I urge the adoption of a quick fix, followed by a more thorough re-design of the email process.
